pkg/server/api_registration_test.go

```diff
@@ -306,6 +319,26 @@ func TestMergeInventoryLabels(t *testing.T) {
 
 func TestUpdateInventoryFromSystemData(t *testing.T) {
 	inventory := &elementalv1.MachineInventory{}
+	registration := &elementalv1.MachineRegistration{
+		Spec: elementalv1.MachineRegistrationSpec{
+			MachineInventoryLabels: map[string]string{
+				"elemental.cattle.io/TotalMemory":            "${System Data/Memory/Total Physical Bytes}",
```

Comment:
@frelon correct me if wrong, but my understanding is that the key name prefix of `elemental.cattle.io` is no longer enforced anywhere right? Here we could have used any other naming criteria, or even no criteria as:

`label1: "${System Data/Memory/Total Physical Bytes}"`